src/add-card-to-project.ts

```diff
@@ -0,0 +1,50 @@
+import { Probot,Context } from 'probot';
+import { LogLevel } from './enums';
+import { log } from './utils/log-util';
+
+export function addCardToProject(probot: Probot) {
+    probot.on("issues.opened", async (context:Context['octokit']) => {
+        try {
+            const { payload, octokit } = context;
+            const repo = payload.repository.name;
+            const owner = payload.sender.login;
+            const issue_number = payload.issue.number;
+
+            const { data: projects } = await octokit.projects.listForRepo({ owner, repo, });
+
+            const project_obj = projects.find(
+                // WARNING: This is just for trial.
+                // TODO: find a way to get issue is associated tags
+                (el:any) => el.name == "x-18-19"
+            );
+                    
+            if (!project_obj) {
+                // TODO: Create new project if it doesn't exist
```

Comment:
We probably wouldn't want to create a new project if it doesn't exist. Logging and bailing is fine:

```
log('addCardToProject', LogLevel.ERROR, `Unable to find project board with name ${name}`);
```
where `name` is the label name that was added to the issue.